test/test_pmm.py

```diff
@@ -729,6 +734,12 @@ def test_config_spread_on_the_fly_multiple_orders(self):
         self.assertAlmostEqual(Decimal("96"), last_bid_order.price, 2)
         self.assertAlmostEqual(Decimal("104"), last_ask_order.price, 2)
 
+    def test_enter_orders_as_array(self):
+        strategy = self.enter_orders_as_array
+        self.assertEqual("buy", strategy.order_override["order1"][0])
+        self.assertEqual(0.5, float(strategy.order_override["order1"][1]))
+        self.assertEqual(100, float(strategy.order_override["order1"][2]))
```

Comment:
The test should cover more scenario including starting the strategy and getting the expected orders submitted.